src/diffusers/models/controlnet_sd3.py

```diff
@@ -375,24 +359,63 @@ def custom_forward(*inputs):
         # 6. scaling
         controlnet_block_res_samples = [sample * conditioning_scale for sample in controlnet_block_res_samples]
 
-
         if USE_PEFT_BACKEND:
             # remove `lora_scale` from each PEFT layer
             unscale_lora_layers(self, lora_scale)
 
         if not return_dict:
-            return (controlnet_block_res_samples, )
```

Comment:
can we revert to`return (controlnet_block_res_samples, )` 
the convention is to return a tuple when `not return_dict`